src/libs/ajax/workspaces/providers/WorkspaceProvider.ts

```diff
@@ -0,0 +1,22 @@
+import { Ajax } from 'src/libs/ajax';
+import { AbortOption } from 'src/libs/ajax/data-provider-common';
+import { WorkspaceWrapper } from 'src/libs/workspace-utils';
+
+export type FieldsArg = string[];
+export type WorkspaceListOptions = AbortOption & {
+  stringAttributeMaxLength?: string | number;
```

Comment:
The API types this parameter as an integer.
https://rawls.dsde-prod.broadinstitute.org/#/workspaces/listWorkspaces

```suggestion
  stringAttributeMaxLength?: number;
```

Reply:
yes, but it appears to be passed in as a string value in the attribute request header.... number will format fine, so I chose the union of both.

Reply:
Sure, everything gets formatted as a string when it's encoded in the URL. But logically, it should be a number so we should use TS to enforce that only a number is provided. Accepting a string would also allow values like `stringAttributeMaxLength: 'notANumber'`.

Reply:
ok yeah, great point.  will change

Reply:
fixed